src/client/src/App.tsx

```diff
@@ -301,6 +307,9 @@ const mapDispatchToProps = (
   updateTemplateGenStatus: (isGenerated: IServiceStatus) => {
     dispatch(updateTemplateGenerationStatusAction(isGenerated));
   },
+  updateDependencyInfo: (dependencyInfo: any) => {
```

Comment:
Just noticed this, can you add the IDependencyInfo in this PR or your refactoring PR? @jsondoo

Reply:
done